src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -0,0 +1,490 @@
+
+                <v-radio-group
+                  v-model="hasNoParValue"
+                  column
+                  class="radio-group"
+                  @change="changeParValueFlag()" v-show="isClass">
+                  <v-radio :value="false" id="radio-par-value">
+                    <template v-slot:label>
+                      <v-row>
+                        <v-col cols="6">
+                          <v-text-field
+                            filled
+                            label="Par Value"
+                            id="class-par-value"
+                            v-model="shareStructure.parValue"
+                            :rules="getParValueRule()"
+                            hint="Enter the initial value of each share"
+                            persistent-hint/>
+                        </v-col>
+                        <v-col cols="6">
+                          <v-select
+                            :items="getCurrencyList()"
+                            filled
+                            label="Currency"
+                            v-model="shareStructure.currency"
+                            :rules="getCurrencyRule()"
+                            item-text="`${data.item.name}, ${data.item.code}`"
+                            item-value="code"
+                            id='class-currency'>
+                            <template slot="selection" slot-scope="data">
+                               {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                            <template slot="item" slot-scope="data">
+                              {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                          </v-select>
+                        </v-col>
+                      </v-row>
+                    </template>
+                  </v-radio>
+                  <v-radio :value="true" label="No par value" id="radio-no-par"/>
+                </v-radio-group>
+
+                <div v-show="isSeries">
+                    <v-row v-if="shareStructure.hasParValue">
+                        <v-col cols="6">
+                            <v-text-field
+                            label="Par Value"
+                            id="series-par-value"
+                            :value="shareStructure.parValue"
+                            :disabled="true"
+                            width="10"/>
+                        </v-col>
+                        <v-col cols="6">
+                            <v-text-field
+                            id="series-currency"
+                            label="Currency"
+                            :value="`${getCurrencyNameByCode(shareStructure.currency)} (${shareStructure.currency})`"
+                            :disabled="true"/>
+                        </v-col>
+                    </v-row>
+                    <v-label id='lbl-no-par' v-else>No par value</v-label>
```

Comment:
Change ' to ".

Reply:
Good catch, thanks! fixed in next commit